mne/decoding/receptive_field.py

```diff
@@ -39,27 +39,33 @@ class ReceptiveField(BaseEstimator):
         float is passed, it will be interpreted as the `alpha` parameter
         to be passed to a Ridge regression model. If `None`, then a Ridge
         regression model with an alpha of 0 will be used.
-    fit_intercept : bool
+    fit_intercept : bool | None
         If True (default), the sample mean is removed before fitting.
-        Ignored if ``estimator`` is a :class:`sklearn.base.BaseEstimator`.
+        If ``estimator`` is a :class:`sklearn.base.BaseEstimator`,
+        this must be None or match ``estimator.fit_intercept``.
     scoring : ['r2', 'corrcoef']
         Defines how predictions will be scored. Currently must be one of
         'r2' (coefficient of determination) or 'corrcoef' (the correlation
         coefficient).
 
     Attributes
     ----------
-    ``coef_`` : array, shape (n_outputs, n_features, n_delays)
+    ``coef_`` : array, shape ([n_outputs, ]n_features, n_delays)
         The coefficients from the model fit, reshaped for easy visualization.
-        If you want the raw (1d) coefficients, access them from the estimator
-        stored in ``self.estimator_``.
+        If ``y`` during :meth:`mne.decoding.ReceptiveField.fit` has one
```

Comment:
I'd say "in the call to :meth:etcetc, if ``y`` has one dimension..."

keeps things less broken-up semantically